Dispatchfile

```diff
@@ -50,5 +61,76 @@ dindTask("dispatch-integration-test",
+                 image="mesosphere/kommander-ci:1045-0bc7580",
+                 workingDir="/workspace/ui-git",
+                 resources=k8s.corev1.ResourceRequirements(
+                     limits={
+                         "cpu": k8s.resource_quantity("8000m"),
+                         "memory": k8s.resource_quantity("6Gi")
+                     }
+                 ),
+                 args=["npm", "install"]),
+
+             v1.Container(
+                 name="install-ui-test-dependencies",
+                 image="mesosphere/kommander-ci:1045-0bc7580",
+                 workingDir="/workspace/ui-git/system-tests",
+                 resources=k8s.corev1.ResourceRequirements(
+                     limits={
+                         "cpu": k8s.resource_quantity("8000m"),
+                         "memory": k8s.resource_quantity("6Gi")
+                     }
+                 ),
+                 args=["npm", "install"]),
+
+             v1.Container(
+                 name="run-e2e-tests",
+                 image="mesosphere/kommander-ci:1045-0bc7580",
+                 workingDir="/workspace/src-git",
+                 resources=k8s.corev1.ResourceRequirements(
+                     limits={
+                         "cpu": k8s.resource_quantity("8000m"),
+                         "memory": k8s.resource_quantity("6Gi")
+                     }
+                 ),
+                 env=[
+                     k8s.corev1.EnvVar(name="DISPATCH_CI", value="true"),
+                     k8s.corev1.EnvVar(name="OUTPUT_PATH",
+                                       value="$(resources.outputs.artifacts.path)"),
+                     k8s.corev1.EnvVar(name="AWS_REGION", value="us-west-2"),
+                     k8s.corev1.EnvVar(name="AWS_ACCESS_KEY",
+                                       valueFrom=secretVar("rotator-ksphere-kommander",
+                                                           "AWS_ACCESS_KEY_ID")),
+                     k8s.corev1.EnvVar(name="AWS_SECRET_KEY",
+                                       valueFrom=secretVar("rotator-ksphere-kommander",
+                                                           "AWS_SECRET_ACCESS_KEY")),
+                     k8s.corev1.EnvVar(name="AWS_ACCESS_KEY_ID",
+                                       valueFrom=secretVar("rotator-ksphere-kommander",
+                                                           "AWS_ACCESS_KEY_ID")),
+                     k8s.corev1.EnvVar(name="AWS_SECRET_ACCESS_KEY",
+                                       valueFrom=secretVar("rotator-ksphere-kommander",
+                                                           "AWS_SECRET_ACCESS_KEY")),
+                     k8s.corev1.EnvVar(name="LICENSE",
+                                       valueFrom=secretVar("kommander-license",
+                                                           "LICENSE"))
+                 ],
+                 command=["bash", "./test/scripts/e2e.sh", "$(context.git.branch)"]),
```

Comment:
It looks like this _always_ runs, I just want to verify that was the intentional _and_ request some additional comments in the PR description to further explain and provide information about what this means to E2E test run times going forward, e.t.c. :thinking:

Reply:
Yeah, the idea was to have them always run so that we can catch problems early on.

Reply:
What kind of information are you looking for?

Reply:
I'll unblock, just curious how long the runs take?

Reply:
They take around 30-45 minutes. Creating the konvoy cluster itself takes as long as running the tests

Reply:
I think there may be a disconnect here. I think Shane means "always" as in even if addons that don't affect the UI are changed where I think Daniel meant "always" in opposition to "nightly".

Reply:
Ah I see. We talked about filtering the tests at some point, but tbh that seems like an over optimization right now. Creating the initial cluster and creating the clusters through kommander take time, testing the addons is currently relatively quick (all in < 5 min)